syntax = "proto3";

package chat;

message User {
  string username = 1;
  string password = 2;
  repeated string room_ids = 3;
}
